lib/moon_web/examples/design/switch_example/for_full_customization.ex

```diff
@@ -0,0 +1,182 @@
+defmodule MoonWeb.Examples.Design.SwitchExample.CodeExampleForFullCustomization do
```

Comment:
i do not see any Switch usage here. what is this example at all?

Reply:
My mistake, misunderstood the example. Please have a look at the new example. However - the positioning of the switcher is not yet custamisable and I would like to ask advice on what would be the best way to solve it.